ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml

```diff
@@ -207,6 +207,19 @@ tests:
     test:
     - chain: openshift-extended-rosacli-tests
     workflow: rosa-aws-sts-shared-vpc-auto
+- as: aws-rosa-sts-shared-vpc-upgrade-e2e-f7
+  cron: 34 4 5,12,19,26 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      AVAILABLE_UPGRADE: "yes"
+      CHANNEL_GROUP: candidate
+      OPENSHIFT_VERSION: 4.14.5
+      UPGRADED_TO_VERSION: "4.15"
+    test:
+    - ref: rosa-cluster-upgrade-cluster
+    - ref: openshift-e2e-test-qe-report
```

Comment:
Looks like we do not need this step since we only run step `rosa-cluster-upgrade-cluster`

Reply:
Removed all `openshift-e2e-test-qe-report` step from upgrade workflow, @yasun1 please confirm the changes, thanks

Reply:
Yes, we do not need it for currently there is no TCs testing on the test chain.